coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -32,7 +32,15 @@
 
 from .._utils import build_einsum_mil, value_at
 from .torch_op_registry import _TORCH_OPS_REGISTRY, register_torch_op
-from .utils import TorchFrontend
+from .utils import (
+    NUM_TO_TORCH_DTYPE,
```

Comment:
nit: it would be nice to alphabetize these entries.

Reply:
Revised as suggested